cfme/tests/physical_infrastructure/test_physical_rack.py

```diff
@@ -27,19 +28,14 @@ def test_physical_rack_details_dropdowns(physical_rack):
     physical_rack.refresh()
 
 
-def physical_rack_collection(appliance, provider, setup_provider_modscope):
-    # Get and return the physical rack collection
-    return appliance.collections.physical_racks
-
-
-def test_physical_racks_view_dropdowns(physical_rack_collection):
+def test_physical_racks_view_dropdowns(appliance, physical_rack):
     """Navigate to the physical racks page and verify that the dropdown menus are present
 
     Polarion:
         assignee: rhcf3_machine
         initialEstimate: 1/4h
     """
-    physical_racks_view = navigate_to(physical_rack_collection, 'All')
+    physical_racks_view = navigate_to(appliance.collections.physical_racks, 'All')
 
     configuration_items = physical_racks_view.toolbar.configuration.items
```

Comment:
This menu item is disabled without a provider configured. This is why I added physical_rack to its parameters list.